Training/configs/training_v1.yaml

```diff
@@ -33,248 +33,371 @@ CellObjectType : [
+                - tau_dz_sig: [*tau_dz_sig_valid, {"tau_dz_sig": "abs(tau_dz)/tau_dz_error"}, "normal", -5, 5]
+                - tau_flightLength_x: [null, null, "normal", -5, 5]
+                - tau_flightLength_y: [null, null, "normal", -5, 5]
+                - tau_flightLength_z: [null, null, "normal", -5, 5]
+                - tau_flightLength_sig: [null, null, "normal", -5, 5]
+                - tau_pt_weighted_deta_strip: [null, null, "linear", 0., 1.]
+                - tau_pt_weighted_dphi_strip: [null, null, "linear", 0., 1.]
+                - tau_pt_weighted_dr_signal: [null, null, "normal", -5, 5]
+                - tau_pt_weighted_dr_iso: [null, null, "linear", 0., 1.]
+                - tau_leadingTrackNormChi2: [null, null, "normal", -5, 5]
+                - tau_e_ratio_valid: [null, null, "no_scaling"]
+                - tau_e_ratio: [null, null, "linear", 0., 1.]
+                - tau_gj_angle_diff_valid: [null, null, "no_scaling"]
+                - tau_gj_angle_diff: [null, null, "linear", 0., *pi]
+                - tau_n_photons: [null, null, "normal", -5, 5]
+                - tau_emFraction: [null, null, "linear", -1., 1.]
+                - tau_inside_ecal_crack: [null, null, "no_scaling"]
+                - tau_leadChargedCand_etaAtEcalEntrance_minus_tau_eta: [null, {"tau_leadChargedCand_etaAtEcalEntrance_minus_tau_eta": "tau_leadChargedCand_etaAtEcalEntrance - tau_eta"}, "normal", -5, 5]
 
-    PfCand_electron : [
-                "pfCand_ele_valid",
-                "pfCand_ele_rel_pt",
-                "pfCand_ele_deta",
-                "pfCand_ele_dphi",
-                "pfCand_ele_pvAssociationQuality",
-                "pfCand_ele_puppiWeight",
-                "pfCand_ele_charge",
-                "pfCand_ele_lostInnerHits",
-                "pfCand_ele_numberOfPixelHits",
-                "pfCand_ele_vertex_dx",
-                "pfCand_ele_vertex_dy",
-                "pfCand_ele_vertex_dz",
-                "pfCand_ele_vertex_dx_tauFL",
-                "pfCand_ele_vertex_dy_tauFL",
-                "pfCand_ele_vertex_dz_tauFL",
-                "pfCand_ele_hasTrackDetails",
-                "pfCand_ele_dxy",
-                "pfCand_ele_dxy_sig",
-                "pfCand_ele_dz",
-                "pfCand_ele_dz_sig",
-                "pfCand_ele_track_chi2_ndof",
-                "pfCand_ele_track_ndof"
-                ]
+    PfCand_electron :
+                - pfCand_ele_valid: [*is_pfElectron, null, "no_scaling"]
+                - pfCand_ele_rel_pt: [*is_pfElectron, {"pfCand_ele_rel_pt": "pfCand_pt/tau_pt"}, "normal", -5, 5]
+                - pfCand_ele_deta: [*is_pfElectron, null, "linear", {'inner': [-0.1, 0.1], 'outer': [-0.5, 0.5]}]
+                - pfCand_ele_dphi: [*is_pfElectron, null, "linear", {'inner': [-0.1, 0.1], 'outer': [-0.5, 0.5]}]
+                - pfCand_ele_pvAssociationQuality: [*is_pfElectron, null, "linear", 0, 7]
```

Comment:
don't scale `pfCand_*_pvAssociationQuality` - it is categorical in nature and we can consider using an embedding layer for it (and other categorical features that could be added in the future).

Reply:
added categorical flag in a062a86